# Project proposal

---

Group name: 

---


In [17]:
import pandas as pd

df = pd.read_csv("../data/raw/ESS5e03_4.csv")

## Introduction

The introduction section includes

-   an introduction to the subject matter you're investigating
-   the motivation for your research question (citing any relevant literature)
-   the general research question you wish to explore
-   your hypotheses regarding the research question of interest.


## Data description

In this section, you will describe the data set you wish to explore. This includes

-   description of the observations in the data set,
-   description of how the data was originally collected (not how you found the data but how the original curator of the data collected it).

### Description of the observations in the data set
Each observation in the data set represents the answers of one participant to the survey questions. The questions can be classified into different subject areas, including e.g. media, politics, family and work or health and well-being. The survey participants were asked various questions on these different topics. The present data set contains the collected answers of all participants. 
This means that each observation contains data on a single person. As only people with German citizenship are to be considered in this project, the data set is reduced to this data. The observations therefore reflect the responses of a German citizen. One column represents one question. The contents of each column are the answers of the respective person to this question.

In [20]:
# first entry of the data set
df.head(1)

name  essround  edition    proddate    idno cntry  dweight   pspwght  \
0  ESS5e03_4         5      3.4  01.12.2018  100102    BE      1.0  0.792865   

    pweight  tvtot  ...  inwyye  inwehh  inwemm  spltadmd  supqad1  supqad2  \
0  0.528619      5  ...  2010.0    14.0     2.0         3        6        1   

   supqdd  supqmm  supqyr  inwtm  
0    20.0    11.0  2010.0   26.0  

[1 rows x 674 columns]

In [23]:
df.shape

(52458, 674)

In [21]:
# data set after filtering for entries with persons of German nationality
df_german = df[df["cntry"]=="DE"]

In [22]:
df_german.shape

(3031, 674)

The data set contains 52458 observations with 674 columns each. The focus on persons with German citizenship reduces the data set under consideration to 3031 observations. 
Likewise, not all columns are to be considered for further analysis. A description of the individual columns is not provided due to the large number. The columns relevant for the analysis are explained in more detail below. Initially, only the categories into which the columns are classified will be listed and briefly explained. 

- **Identifier and weight variables**:  information about the data set and survey like the title of the data set, production date or country of the respondent
- **Media and social trust**:   information about media consumption (such as TV, radio, Internet) and trust in strangers
- **Politics**:     information about the trust in politics, the party voted for in the last election and the satisfaction with political system in the respective country
- **Subjective well-being, social exclusion, religion, national and ethnic identity**:  information about the well-being in different situations, social behavior and nationality
- **Family work and well-being**:   information about own job and their partner's job, for e.g. job security, pay and satisfaction
- **Justic**:   information about overall crime situation in their country, police and the law
- **Gender, Year of birth and Household grid**:     information about the gender and age of the respondent as well as the age and relationship to members in their household
- **Socio-demographics**:   information about relationship status, education and income of the respondent, their partner and parents
- **Human values**:     information about a person's attitude to how they live their life like priorities in life, treat other people and want to be treated
- **Administrative variables**:     information about the interview like place or date


## Analysis approach

In this section, you will provide a brief overview of your analysis approach. This includes:

-   Description of the response variable.
-   Visualization and summary statistics for the response variable.
-   List of variables that will be considered as predictors
-   Your model type (what kind of model(s) will you use ... e.g. linear regression)

## Data dictionary

*Create a data dictionary for all the variables in your data set. You may fill out the data description table or create your own table with Pandas:*

<br>


| Name  |   Description	| Role   	| Type   	|  Format 	|
|---	|---	        |---    	|---	    |---	|
|   	|   	        |   	    |   	    |   	|
|   	|   	        |   	    |   	    |   	|
|   	|   	        |   	    |   	    |   	|


<br>

- `Role`: response, predictor, ID (ID columns are not used in a model but can help to better understand the data)

- `Type`: nominal, ordinal or numeric

- `Format`: int, float, string, category, date or object